packages/core/test/controls/numericInputTests.tsx

```diff
@@ -432,6 +467,30 @@ describe("<NumericInput>", () => {
                 const newValue = component.state().value;
                 expect(newValue).to.equal(MAX_VALUE.toString());
             });
+
+            it("fires onValueChange with clamped value if nextProps.max < value ", () => {
```

Comment:
wait why is it fired at all if the value didn't actually change? this sounds like what we just went through with `EditableText` #629